Gemfile

```diff
@@ -2,6 +2,7 @@ source "https://rubygems.org"
 
 gemspec
 
+gem "avro_turf"
```

Comment:
As this is a gem whatever you put in a Gemfile will not be installed when you put `gem 'glass_octopus'` in your application's Gemfile.

I think this is the right approach. `avro_turf` is an optional dependency so we should not impose it on every user when they don't need it.

Optional dependencies should be mentioned in the README.